docs/ddl-operation-on-carbondata.md

```diff
@@ -115,8 +115,7 @@ TBLPROPERTIES ("COLUMN_GROUPS"="(column1,column3),
                                 saleQuantity Int,
                                 revenue Int)       
    STORED BY 'carbondata' 
-   TBLPROPERTIES ('COLUMN_GROUPS'='(productName,productCategory)',
-                  'DICTIONARY_EXCLUDE'='productName',
+   TBLPROPERTIES ('COLUMN_GROUPS'='(productNumber,productName)',
                   'DICTIONARY_INCLUDE'='productNumber',
```

Comment:
Please add "DICTIONARY_EXCLUDE" with other column, which doesn't belong to COLUMN_GROUPS.
Because this example is introducing both (DICTIONARY_EXCLUDE and DICTIONARY_INCLUDE)